Vagrantfile

```diff
@@ -307,4 +307,9 @@ Defaults   env_keep += "BATS_ARCHIVES"
 SUDOERS_VARS
     chmod 0440 /etc/sudoers.d/elasticsearch_vars
   SHELL
+  # This prevents leftovers from previous tests using the
+  # same VM from messing up the current test
+  config.vm.provision "clean_tmp", run: "always", type: "shell", inline: <<-SHELL
+    rm -rf /tmp/elastic*earch*
```

Comment:
Since you fixed the missing `s`, can we leave this simply as `/tmp/elasticsearch`? The VMs on CI will eventually be cleaned up when, for example, the boxes are updated?

Reply:
Sure, I've changed this